composer.json

```diff
@@ -73,28 +73,26 @@
     "require-dev": {
         "akeneo/phpspec-skip-example-extension": "^3.0",
         "behat/behat": "^3.2",
-        "behat/mink": "^1.7",
-        "behat/mink-browserkit-driver": "^1.3",
+        "behat/mink": "^1.7@dev",
+        "behat/mink-browserkit-driver": "^1.3@dev",
         "behat/mink-extension": "^2.2",
         "behat/mink-selenium2-driver": "^1.3",
-        "friends-of-behat/context-service-extension": "^1.0",
-        "friends-of-behat/cross-container-extension": "^1.0",
+        "friends-of-behat/context-service-extension": "^1.2",
+        "friends-of-behat/cross-container-extension": "^1.1",
         "friends-of-behat/service-container-extension": "^1.0",
-        "friends-of-behat/symfony-extension": "^1.0",
-        "friends-of-behat/variadic-extension": "^1.0",
-        "hwi/oauth-bundle": "^0.5",
-        "lakion/api-test-case": "^1.0",
+        "friends-of-behat/symfony-extension": "^1.1",
+        "friends-of-behat/variadic-extension": "^1.1",
+        "hwi/oauth-bundle": "^0.6",
+        "lakion/api-test-case": "dev-test-matrix-coduo",
```

Comment:
3.0 release should be ready soon**ish** and it probably would include Lakion/ApiTestCase#111.